dcspy/qt_gui.py

```diff
@@ -661,7 +672,14 @@ def _input_iface_changed_or_custom_text_changed(self) -> None:
             ctrl_key = self.ctrl_input[section][current_cell_text]
             custom_value = ''
             if (input_iface_name := self.bg_rb_input_iface.checkedButton().objectName()) == 'rb_custom':
-                custom_value = self.le_custom.text() if self.le_custom.text()[-1] == '|' else f'{self.le_custom.text()}|'
+                # if there is no custom text and this is a push_button then default to BUTTON,
```

Comment:
This is temporary and intended to be replaced with an actual radio button in later commits.

Reply:
Probably we need another if for BUTTON.... start to looking awful... but we can refactor when we got working solution.